### PR TITLE
Enable win

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by ryanvolz** [comment]
This first build didn't use the new libusb package. Give it another try now or restart the CI.

**Comment by ryanvolz** [comment]
Ah, `libftdi` doesn't really use pkg-config to find `libusb`, it just uses it as a hint:

http://developer.intra2net.com/git/?p=libftdi;a=blob;f=cmake/FindUSB1.cmake;h=b90e297c7facdce6adf0bfc24f9c8e82eb4e781f;hb=HEAD

You'll need to patch that file to include `libusb-1.0` as an alternate name (after `usb-1.0`) or modify the Windows build script to tell CMake the header and library path explicitly.

**Comment by Tobias-Fischer** [comment]
@ryanvolz:

Even with pure pkg-config it doesn't work:
```
Checking for module 'libusb-1.0'
--   No package 'libusb-1.0' found
```
PS: ignore the build fails on Unix, I'll fix these tomorrow - wrong naming of variables in this case.

**Comment by ryanvolz** [comment]
I did some digging, and I re-discovered that `pkg-config` on Windows only sees things in the environment that it is installed into by default, in this case the build environment. So it misses anything in host, including `libusb`. Here's what you need to add to your `bld.bat` to get `pkg-config` to look everywhere:
```
:: set pkg-config path so that host deps can be found
set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig"
```
Stick that in before you call `cmake`. *Then* it should be able to find `libusb-1.0.pc` and we'll finally be able to find out if it actually works for the build. Alternatively there's always the CMake tweaks that get around `pkg-config` completely: https://github.com/conda-forge/libftdi-feedstock/pull/1#issuecomment-824529044. As you can tell, not many people/projects really use `pkg-config` on Windows, at least when building with MSVC.

**Comment by Tobias-Fischer** [comment]
Thanks @ryanvolz - this worked; and it's compiling fine now.

Question - no ".lib" is created, only a ".dll". Is that an issue? @wolfv @traversaro - you guys know much more than I do :)

**Comment by ryanvolz** [comment]
I looks like that happens when nothing is declared with `dllexport`: https://stackoverflow.com/questions/64088046/missing-lib-file-when-creating-shared-library-with-cmake-and-visual-studio-2019. I don't know if that's intended by upstream, but probably not? My guess is that not many people have tried building/using this with MSVC, e.g. http://developer.intra2net.com/mailarchive/html/libftdi/2021/msg00008.html.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,10 +28,13 @@ requirements:
...
-    - test -f ${PREFIX}/include/libftdi1/ftdi.h  # [unix]
-    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}  # [unix]
-    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc  # [unix]
+    - test -f ${PREFIX}/include/libftdi1/ftdi.h                  # [unix]
+    - test -f ${PREFIX}/lib/libftdi1${SHLIB_EXT}                 # [unix]
+    - test -f ${PREFIX}/lib/pkgconfig/libftdi1.pc                # [unix]
     - test -f ${PREFIX}/lib/cmake/libftdi1/LibFTDI1Config.cmake  # [unix]
+    - if not exist %LIBRARY_INC%\libftdi1\ftdi.h (exit 1)        # [win]
+    # - if not exist %LIBRARY_LIB%\libftdi1.lib (exit 1)           # [win]
+    - if not exist %LIBRARY_BIN%\libftdi1.dll (exit 1)           # [win]
```
Comment (ryanvolz) [inline recipe/meta.yaml]:
```suggestion
    - if not exist %LIBRARY_LIB%\ftdi1.lib (exit 1)           # [win]
    - if not exist %LIBRARY_BIN%\ftdi1.dll (exit 1)           # [win]
```
Windows doesn't put 'lib' in front of libraries by default, so this is what they will be called.

**Comment by Tobias-Fischer** [comment]
Many thanks for your help @ryanvolz

**Pull request merged**